frontend/components/PaginatedClubDisplay.tsx

```diff
@@ -98,6 +100,7 @@ const PaginatedClubDisplay = ({
           tags={tags}
           display={display}
           onScroll={loadNextPage}
+          ranked={ranked}
```

Comment:
Is this....
![image](https://github.com/pennlabs/penn-clubs/assets/62971511/1e1a36f4-d928-472a-98bf-520ee5c4805e)

Reply:
![61ZkaCujGrL _AC_UF1000,1000_QL80_](https://github.com/pennlabs/penn-clubs/assets/10285518/b4fa93bc-4592-44d4-a65e-aaa80e411a19)